tools/mo/requirements_dev.txt

```diff
@@ -1,6 +1,6 @@
 coverage>=4.4.2,<=5.5
 astroid==2.4.2
-pylint==2.5.0
+pylint==2.7.0
```

Comment:
You need to loosen requirements. Please only restrict them "from the bottom" like:
```
astroid>=2.5.0
pylint>=2.7.0
```
Note: Look at astroid packaged being bumped as well, this also fix linux error with deps